partiql-types/README.adoc

```diff
@@ -1,317 +0,0 @@
-= PartiQL Types & Values
```

Comment:
Should this be copied over to spi as well?

Reply:
Eh, it's largely out of date with the use of PartiQLValue. Also, documentation for types will be on the website, especially with the most recent contribution.